packages/nouns-webapp/src/config.ts

```diff
@@ -90,30 +93,42 @@ const externalAddresses: Record<SupportedChains, ExternalContractAddresses> = {
   [ChainId.Rinkeby]: {
     lidoToken: '0xF4242f9d78DB7218Ad72Ee3aE14469DBDE8731eD',
     usdcToken: '0xeb8f08a975Ab53E34D8a0330E0D34de942C95926',
+    weth: undefined,
     payerContract: undefined,
     tokenBuyer: undefined,
     chainlinkEthUsdc: undefined,
+    nounsStream: undefined,
+    nounsStreamFactory: undefined,
   },
   [ChainId.Goerli]: {
     lidoToken: '0x2DD6530F136D2B56330792D46aF959D9EA62E276',
     usdcToken: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
+    weth: '0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6',
     payerContract: '0xD4A3bf1dF54699E63A2ef7F490E8E22b27B945f0',
     tokenBuyer: '0x61Ec4584c5B5eBaaD9f21Aac491fBB5B2ff30779',
     chainlinkEthUsdc: undefined,
+    nounsStream: '0xb82c6b4889969deba04d2ade4687a4656186895d',
```

Comment:
@bcjgit I think this is unused, can be removed. the frontend should only need to interact with the StreamFactory or new stream clones